data/elevation-terrain-data/2013-2014-lidar/index.html

```diff
@@ -121,20 +121,13 @@
   return number, RGB value etc. Each LiDAR point can also have classifications for landcover types (ex. ground, water, road, building, vegetation, etc.).
   The LAS point cloud files associated with this project have the following classifications yet further classification of the point cloud is possible:
     <dl>
-        <dt>Code</dt>
-        <dd>Description</dd>
-        <dt>1</dt>
-        <dd>Processed, but unclassified</dd>
-        <dt>2</dt>
-        <dd>Bare-earth ground</dd>
-        <dt>7</dt>
-        <dd>Noise</dd>
-        <dt>9</dt>
-        <dd>Water</dd>
-        <dt>10</dt>
-        <dd>Ignored Ground (Breakline proximity)</dd>
-        <dt>11</dt>
-        <dd>Withheld</dd>
+      <dt>Code | Description</dt>
```

Comment:
this isn't the semantically correct use of a definition list. We should probably replace this with something more appropriate.